downstream/assemblies/platform/assembly-mesh-design-patterns.adoc

```diff
@@ -1,13 +1,13 @@
 
 ifdef::context[:parent-context: {context}]
 
-[id="assembly-standalone-controller-non-inst-database"]
+[id="design-patterns"]
 = {AutomationMeshStart} design patterns
 
-:context: standalone-controller-non-inst-database
+:context: design-patterns
 
 [role="_abstract"]
-The automation mesh topologies in this section are intended to provide examples you can use to design a mesh deployment in your environment. Examples range from a single, hydrid node deployment to a complex pattern that deploys numerous {ControllerName}s, employing several execution and hop nodes.
+The automation mesh topologies in this section provide examples you can use to design a mesh deployment in your environment. Examples range from a single, hydrid node deployment to a complex pattern that deploys numerous {ControllerName}s, employing several execution and hop nodes.
```

Comment:
```suggestion
The automation mesh topologies in this section provide examples you can use to design a mesh deployment in your environment. Examples range from a single, hydrid node deployment to a complex pattern that deploys numerous {ControllerName} instances, employing several execution and hop nodes.
```

Reply:
Final nit here but I think it's never a good idea to tack an `s` on the end of a variable to pluralize it. Doing that is kind of a hack and can go wrong.

Either add a plural noun after it - like "instances", "deployments", "installations", "clusters" or whatever might be appropriate - or create a plural version of the corresponding variable.